coordinator/src/main/scala/filodb.coordinator/queryplanner/MultiPartitionPlanner.scala

```diff
@@ -96,7 +98,12 @@ class MultiPartitionPlanner(partitionLocationProvider: PartitionLocationProvider
     } else logicalPlan match {
       case mqp: MetadataQueryPlan             => materializeMetadataQueryPlan(mqp, qContext).plans.head
       case lp: TsCardinalities                => materializeTsCardinalities(lp, qContext).plans.head
-      case _                                  => walkLogicalPlanTree(logicalPlan, qContext).plans.head
+      case _                                  =>
+        val result = walkLogicalPlanTree(logicalPlan, qContext)
```

Comment:
Good point -- this was probably needed for some previous PR iteration, but tests pass without it, now. I will drop this change since the extra logic isn't necessary.